CMakeLists.txt

```diff
@@ -131,21 +131,44 @@ option( BUILD_SHARED_LIBS "Build rocBLAS as a shared library" ON )
 include( clients/cmake/build-options.cmake )
 
 if( BUILD_WITH_TENSILE )
-  include(virtualenv)
   set( Tensile_LOGIC "asm_full" CACHE STRING "Tensile to use which logic?")
-  set_property( CACHE Tensile_LOGIC PROPERTY STRINGS asm_full asm_lite hip_lite other )
 
   option( Tensile_MERGE_FILES "Tensile to merge kernels and solutions files?" ON )
   option( Tensile_SHORT_FILENAMES "Tensile to use short file names? Use if compiler complains they're too long." OFF )
   option( Tensile_PRINT_DEBUG "Tensile to print runtime debug info?" OFF )
 
-  set( tensile_tag "develop" CACHE STRING "Tensile tag to download" )
-  virtualenv_install("git+https://github.com/ROCmSoftwarePlatform/Tensile.git@${tensile_tag}")
-  list(APPEND CMAKE_PREFIX_PATH ${VIRTUALENV_HOME_DIR})
+  set( Tensile_TEST_LOCAL_PATH "" CACHE PATH "Use local Tensile directory instead of fetching a GitHub branch" )
+  set( CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended onto install directories" )
+  set_property( CACHE Tensile_LOGIC PROPERTY STRINGS asm_full asm_lite hip_lite other )
+
+
+  if (Tensile_TEST_LOCAL_PATH)
+    # Use user's local Tensile path:
+    execute_process( COMMAND ${CMAKE_COMMAND} -E copy_directory ${Tensile_TEST_LOCAL_PATH} ${PROJECT_EXTERN_DIR}/Tensile)
+
+    #set( Tensile_ROOT ${Tensile_TEST_LOCAL_PATH} )
+    set( Tensile_ROOT "${PROJECT_EXTERN_DIR}/Tensile" CACHE STRING "Local path of Tensile" )
+    set( Tensile_TensileConfig ${Tensile_ROOT}/Tensile/Source/TensileConfig.cmake )
+  
+    message (STATUS "using local Tensile from ${Tensile_TEST_LOCAL_PATH}, copied to ${Tensile_ROOT}")
+  else()
+    # Use the virtual-env setup and download package from specified repot:
+    include(virtualenv)
+
+    set( tensile_fork "ROCmSoftwarePlatform" CACHE STRING "Tensile fork to use" )
+    set( tensile_branch "develop" CACHE STRING "Tensile tag to download" )
```

Comment:
Can we make the default    master   . The reasoning is that before releasing the develop branch of rocBLAS is merged into master. We do not want to have to change this before the merge. It is dangerous if we forget to change it, as then the master branch of rocBLAS will be pulling the develop branch of Tensile.

Reply:
Actually, I think this should use a commit hash or a tag as the default. Otherwise, we wont have reproducible builds for rocblas.